static-metric/examples/make_auto_flush_static_metric_histo.rs

```diff
@@ -0,0 +1,432 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Could you rename the file to `make_auto_flush_static_histogram.rs`? I believe `histo` is a confusing abbreviation.